### ISSUE TITLE
Spike: Analyze Unity Build size and look for potential savings

### ISSUE THREAD

**Issue by eordano** [decentraland/unity-renderer#3874]
Acceptance criteria:
- [] pie chart or table with % of size
- [] insights about candidate resources to lazy load

**Comment by eordano** [comment]
```
Build Report
Uncompressed usage by category (Percentages based on user generated assets only):
Textures               90.6 mb	 50.1% 
Meshes                 805.1 kb	 0.4% 
Animations             53.4 mb	 29.5% 
Sounds                 4.7 mb	 2.6% 
Shaders                6.0 mb	 3.3% 
Other Assets           23.7 mb	 13.1% 
Levels                 0.0 kb	 0.0% 
Scripts                561.0 kb	 0.3% 
Included DLLs          0.0 kb	 0.0% 
File headers           1.2 mb	 0.7% 
Total User Assets      181.1 mb	 100.0% 
Complete build size    167.5 mb
```

**Comment by eordano** [comment]
Assets weighting more than 100kb:

```
-------------------------------------------------------------------------------
Build Report
Uncompressed usage by category (Percentages based on user generated assets only):
Textures               90.6 mb	 50.1% 
Meshes                 805.1 kb	 0.4% 
Animations             53.4 mb	 29.5% 
Sounds                 4.7 mb	 2.6% 
Shaders                6.0 mb	 3.3% 
Other Assets           23.7 mb	 13.1% 
Levels                 0.0 kb	 0.0% 
Scripts                561.0 kb	 0.3% 
Included DLLs          0.0 kb	 0.0% 
File headers           1.2 mb	 0.7% 
Total User Assets      181.1 mb	 100.0% 
Complete build size    167.5 mb
Used Assets and files from the Resources folder, sorted by uncompressed size:
 16.0 mb	 9.6% Built-in Texture2D: sactx-0-4096x4096-DXT5|BC3-TexturesUI-ed4b4f81
 8.1 mb	 4.8% Assets/TextMesh Pro/Resources/Fonts Sources/NotoSansSC-SemiBold.otf
 8.0 mb	 4.8% Assets/Rendering/ProceduralSkybox/ResourcesProceduralSkybox/Textures/Default_Skybox_NearClouds_2.png
 8.0 mb	 4.8% Assets/Rendering/ProceduralSkybox/ResourcesProceduralSkybox/Textures/Default_Skybox_FarClouds.png
 8.0 mb	 4.8% Assets/Rendering/ProceduralSkybox/ResourcesProceduralSkybox/Textures/CubemapPlazaCut.png
 8.0 mb	 4.8% Assets/Rendering/ProceduralSkybox/ResourcesProceduralSkybox/Textures/mountains_01.png
 6.1 mb	 3.7% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/DanceHipHop.anim
 4.5 mb	 2.7% Assets/TextMesh Pro/Resources/Fonts Sources/NotoSansKR-SemiBold.otf
 4.4 mb	 2.6% Assets/Scripts/MainScripts/DCL/Controllers/HUD/SignupHUD/Textures/SignUpAvatars.png
 4.4 mb	 2.6% Assets/TextMesh Pro/Resources/Fonts Sources/NotoSansJP-SemiBold.otf
 4.0 mb	 2.4% Assets/Scripts/MainScripts/DCL/Controllers/HUD/QuestsPanelHUD/Textures/Rays.png
 4.0 mb	 2.4% Built-in Texture2D: sactx-0-2048x2048-DXT5|BC3-LoadingAtlas-f0bce718
 4.0 mb	 2.4% Assets/Textures/LandGrid.png
 3.5 mb	 2.1% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/robot.anim
 2.7 mb	 1.6% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/robot.anim
 1.9 mb	 1.1% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/clap.anim
 1.9 mb	 1.1% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/clap.anim
 1.7 mb	 1.0% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/money.anim
 1.7 mb	 1.0% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/money.anim
 1.7 mb	 1.0% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/kiss.anim
 1.7 mb	 1.0% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/kiss.anim
 1.5 mb	 0.9% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/dance.anim
 1.5 mb	 0.9% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/dance.anim
 1.5 mb	 0.9% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/hammer.anim
 1.5 mb	 0.9% Assets/Scripts/MainScripts/DCL/Components/Avatar/Resources/Textures/AvatarParticleExpressionHeartPride.png
 1.4 mb	 0.8% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/tektonik.anim
 1.4 mb	 0.8% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/tik.anim
 1.4 mb	 0.8% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/disco.anim
 1.4 mb	 0.8% Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Resources/Music/Music - Avatar Editor HUD - A.ogg
 1.3 mb	 0.8% Assets/Rendering/ProceduralSkybox/ResourcesProceduralSkybox/Textures/T_Stars01.png
 1.1 mb	 0.7% Assets/Tutorial/Music/Music - Tutorial.ogg
 1.1 mb	 0.6% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/Idle.anim
 1.0 mb	 0.6% Assets/TextMesh Pro/Resources/Fonts & Materials/LiberationSans SDF.asset
 1.0 mb	 0.6% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/snowfall.anim
 1.0 mb	 0.6% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/snowfall.anim
 1.0 mb	 0.6% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/fistpump.anim
 1.0 mb	 0.6% Built-in Texture2D: sactx-0-512x512-Uncompressed-TutorialStepsAtlas-68367ae6
 1.0 mb	 0.6% Built-in Texture2D: sactx-0-1024x1024-DXT5|BC3-AvatarEditorAtlas-e9919969
 1.0 mb	 0.6% Assets/Rendering/InfiniteFloor/Resources/MapDefaultEstates.png
 1.0 mb	 0.6% Assets/Scripts/MainScripts/DCL/Controllers/HUD/QuestsPanelHUD/Textures/PoapDefault.png
 1.0 mb	 0.6% Assets/Rendering/ProceduralSkybox/ResourcesProceduralSkybox/Textures/T_Clouds01.png
 1.0 mb	 0.6% Assets/Rendering/InfiniteFloor/Resources/MapDefault.png
 1.0 mb	 0.6% Assets/Rendering/ProceduralSkybox/ResourcesProceduralSkybox/Textures/T_Sun.png
 1.0 mb	 0.6% Assets/Rendering/ProceduralSkybox/ResourcesProceduralSkybox/Textures/T_Star01.png
 1.0 mb	 0.6% Assets/Rendering/ProceduralSkybox/ResourcesProceduralSkybox/Textures/T_Moon.png
 1024.0 kb	 0.6% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/wave.anim
 1023.6 kb	 0.6% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/wave.anim
 891.2 kb	 0.5% Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Textures/DefaultImagePlace.png
 890.0 kb	 0.5% Packages/com.decentraland.unity-shared-dependencies/Runtime/Shaders/URP/Lit.shader
 879.8 kb	 0.5% Assets/Rendering/Shaders/Toon/Compiled/ToonShaderCompiled.shader
 833.9 kb	 0.5% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/fistpump.anim
 779.1 kb	 0.5% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Victory.anim
 768.1 kb	 0.4% Assets/Rendering/Shaders/Toon/Gradients/Final MatCap.psd
 756.9 kb	 0.4% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/handsair.anim
 683.1 kb	 0.4% Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/NFTShape/Textures/MulticolorDotsLoading.png
 589.8 kb	 0.3% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/headexplode.anim
 589.8 kb	 0.3% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/headexplode.anim
 576.5 kb	 0.3% Assets/Scripts/MainScripts/DCL/Controllers/HUD/SignupHUD/Textures/Background.jpeg
 567.0 kb	 0.3% Assets/UI Toolkit/UnityThemes/UnityDefaultRuntimeTheme.tss
 546.5 kb	 0.3% Assets/Fonts/Monaco.dfont
 520.8 kb	 0.3% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/BackpackAnimations/Lower/Animations/Outfit_Lower_v02.anim
 512.2 kb	 0.3% Built-in Texture2D: sactx-0-512x1024-DXT5|BC3-TeleportLoadingAtlas-2cb3f9ed
 494.1 kb	 0.3% Assets/Scripts/MainScripts/DCL/Controllers/HUD/TeleportPromptHUD/Textures/Magic.jpg
 494.0 kb	 0.3% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/crafting.anim
 494.0 kb	 0.3% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/crafting.anim
 482.4 kb	 0.3% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/hohoho.anim
 482.4 kb	 0.3% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/hohoho.anim
 480.6 kb	 0.3% Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Textures/DefaultImageEvent.png
 476.4 kb	 0.3% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/BackpackAnimations/Upper/Animations/Outfit_Upper_v02.anim
 466.0 kb	 0.3% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/raiseHand.anim
 466.0 kb	 0.3% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/raiseHand.anim
 446.7 kb	 0.3% Assets/UnityGLTF/Shaders/PbrSpecularGlossiness.shader
 438.4 kb	 0.3% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/BackpackAnimations/Upper/Animations/Outfit_Upper_v01.anim
 415.3 kb	 0.2% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/BackpackAnimations/Lower/Animations/Outfit_Lower_v01.anim
 406.1 kb	 0.2% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/dontsee.anim
 404.8 kb	 0.2% Packages/com.unity.render-pipelines.universal/Shaders/Lit.shader
 399.6 kb	 0.2% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/Walk.anim
 395.1 kb	 0.2% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/shrug.anim
 392.6 kb	 0.2% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/BackpackAnimations/Shoes/Animations/Outfit_Shoes_v01.anim
 389.3 kb	 0.2% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/BackpackAnimations/Lower/Animations/Outfit_Lower_v03.anim
 384.4 kb	 0.2% Assets/Rendering/Shaders/URP/UberPost.shader
 367.9 kb	 0.2% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/Idle.anim
 348.8 kb	 0.2% Packages/com.unity.render-pipelines.universal/Shaders/SimpleLit.shader
 348.7 kb	 0.2% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/dab.anim
 348.7 kb	 0.2% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/dab.anim
 347.6 kb	 0.2% Packages/com.unity.render-pipelines.universal/Shaders/PostProcessing/UberPost.shader
 344.5 kb	 0.2% Assets/TextMesh Pro/Fonts/LiberationSans.ttf
 341.5 kb	 0.2% Assets/TextMesh Pro/Sprites/EmojiOne.png
 323.0 kb	 0.2% Assets/Scripts/MainScripts/DCL/Controllers/HUD/TeleportPromptHUD/Textures/crowd.jpg
 321.2 kb	 0.2% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/BackpackAnimations/Shoes/Animations/Outfit_Shoes_v02.anim
 320.7 kb	 0.2% Assets/UnityGLTF/Shaders/PbrMetallicRoughness.shader
 309.0 kb	 0.2% Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Resources/MainMenu/ExploreV2Menu.prefab
 304.4 kb	 0.2% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Waving.anim
 285.9 kb	 0.2% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/Run.anim
 283.4 kb	 0.2% Assets/TextMesh Pro/Resources/Fonts & Materials/Inter-Heavy SDF.asset
 283.2 kb	 0.2% Assets/TextMesh Pro/Resources/Fonts & Materials/Inter-Bold SDF.asset
 282.9 kb	 0.2% Assets/TextMesh Pro/Resources/Fonts & Materials/Inter-Regular SDF.asset
 282.8 kb	 0.2% Assets/TextMesh Pro/Resources/Fonts & Materials/Inter-SemiBold SDF.asset
 278.8 kb	 0.2% Assets/TextMesh Pro/Resources/Fonts & Materials/SF-UI-Text-Regular SDF.asset
 278.7 kb	 0.2% Assets/TextMesh Pro/Resources/Fonts & Materials/SF-UI-Text-Heavy SDF.asset
 278.5 kb	 0.2% Assets/TextMesh Pro/Resources/Fonts & Materials/SF-UI-Text-Semibold SDF.asset
 276.1 kb	 0.2% Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Resources/AvatarEditorHUD.prefab
 274.5 kb	 0.2% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/Run.anim
 274.2 kb	 0.2% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/Jump.anim
 274.2 kb	 0.2% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/Fall.anim
 274.2 kb	 0.2% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/Jump.anim
 274.2 kb	 0.2% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/Fall.anim
 267.6 kb	 0.2% Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Resources/Sections/PlacesAndEventsSection/PlacesAndEventsSection.prefab
 262.6 kb	 0.2% Packages/com.unity.render-pipelines.universal/Textures/SMAA/AreaTex.tga
 258.9 kb	 0.2% Assets/TextMesh Pro/Resources/Fonts & Materials/NotoSansJP-SemiBold SDF.asset
 256.1 kb	 0.1% Assets/Scripts/MainScripts/DCL/WorldRuntime/BlockerController/Resources/LoadingBlocker_Purple_Base.psd
 256.1 kb	 0.1% Assets/Scripts/MainScripts/DCL/WorldRuntime/BlockerController/Resources/LoadingBlocker_Green_Base.psd
 256.1 kb	 0.1% Assets/Resources/Textures/avatar-impostors-atlas.png
 256.1 kb	 0.1% Packages/com.unity.render-pipelines.universal/Textures/FilmGrain/Thin02.png
 256.1 kb	 0.1% Packages/com.unity.render-pipelines.universal/Textures/FilmGrain/Thin01.png
 256.1 kb	 0.1% Packages/com.unity.render-pipelines.universal/Textures/FilmGrain/Medium06.png
 256.1 kb	 0.1% Packages/com.unity.render-pipelines.universal/Textures/FilmGrain/Medium05.png
 256.1 kb	 0.1% Packages/com.unity.render-pipelines.universal/Textures/FilmGrain/Medium04.png
 256.1 kb	 0.1% Packages/com.unity.render-pipelines.universal/Textures/FilmGrain/Medium03.png
 256.1 kb	 0.1% Packages/com.unity.render-pipelines.universal/Textures/FilmGrain/Medium02.png
 256.1 kb	 0.1% Packages/com.unity.render-pipelines.universal/Textures/FilmGrain/Medium01.png
 256.1 kb	 0.1% Packages/com.unity.render-pipelines.universal/Textures/FilmGrain/Large02.png
 256.1 kb	 0.1% Packages/com.unity.render-pipelines.universal/Textures/FilmGrain/Large01.png
 256.1 kb	 0.1% Assets/Textures/ground.jpg
 256.1 kb	 0.1% Assets/Rendering/ProceduralSkybox/ResourcesProceduralSkybox/Textures/Distort.bmp
 246.4 kb	 0.1% Assets/Builder/Gizmos/RotateGizmo.fbx
 242.1 kb	 0.1% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/BackpackAnimations/Upper/Animations/Outfit_Upper_v03.anim
 235.2 kb	 0.1% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/BackpackAnimations/Accessories/Animations/Outfit_Accessories_v02.anim
 233.6 kb	 0.1% Assets/Textures/UI/Emotes/WheelSlot.png
 231.0 kb	 0.1% Resources/unity_builtin_extra
 202.3 kb	 0.1% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/BackpackAnimations/Accessories/Animations/Outfit_Accessories_v03.anim
 175.7 kb	 0.1% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/snowballthrow.anim
 175.7 kb	 0.1% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/snowballthrow.anim
 172.5 kb	 0.1% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/BackpackAnimations/Spawn/Animations/Spawn_Pose_v01.anim
 170.8 kb	 0.1% Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/Textures/MinimapPlaceholder.png
 170.8 kb	 0.1% Assets/Rendering/InfiniteFloor/Grass_Grayscale.png
 170.8 kb	 0.1% Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/NFTShape/Textures/Genesis_TX.png
 170.8 kb	 0.1% Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/NFTShape/Textures/Floor_Wood01.png
 161.9 kb	 0.1% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Male/snowballhit.anim
 161.9 kb	 0.1% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/snowballhit.anim
 145.0 kb	 0.1% Assets/Rendering/LoadingAvatar/CrossSection/Avatar_Male_SingleMesh.glb
 142.3 kb	 0.1% Assets/Tutorial/Prefabs/Common/TutorialRobot/Model/AliceRobot.fbx
 141.5 kb	 0.1% Packages/com.atteneder.gltfast/Runtime/Shader/Legacy/glTF-pbrMetallicRoughness-Opaque-double.shadergraph
 140.1 kb	 0.1% Packages/com.atteneder.gltfast/Runtime/Shader/Legacy/glTF-pbrMetallicRoughness-Opaque.shadergraph
 136.3 kb	 0.1% Packages/com.unity.render-pipelines.universal/Shaders/Terrain/TerrainLit.shader
 129.1 kb	 0.1% Assets/Rendering/InfiniteFloor/InfiniteFloorShaders/S_infiniteFloor_r_1_7F.shader
 128.1 kb	 0.1% Assets/Rendering/ProceduralSkybox/ResourcesProceduralSkybox/Textures/T_NormalsWithAlphaNoise.png
 128.1 kb	 0.1% Assets/Scripts/MainScripts/DCL/WorldRuntime/BlockerController/Resources/Gradient.psd
 128.0 kb	 0.1% Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Resources/Sections/PlacesAndEventsSection/EventsSubSection/EventsSubSection.prefab
 127.2 kb	 0.1% Packages/com.atteneder.gltfast/Runtime/Shader/Legacy/glTF-pbrMetallicRoughness-Blend-double.shadergraph
 127.2 kb	 0.1% Packages/com.atteneder.gltfast/Runtime/Shader/Legacy/glTF-pbrMetallicRoughness-Blend.shadergraph
 117.5 kb	 0.1% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/BackpackAnimations/Accessories/Animations/Outfit_Accessories_v01.anim
 112.8 kb	 0.1% Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Resources/EmotesCustomization/EmotesCustomizationSection.prefab
 112.5 kb	 0.1% Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/Resources/Female/Walk.anim
 101.6 kb	 0.1% Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Resources/Sections/PlacesAndEventsSection/HighlightsSubSection/HighlightsSubSection.prefab
```

**Comment by eordano** [comment]
https://github.com/decentraland/unity-renderer/issues/3878 was created from this
